packages/camera/camera_android_camerax/lib/src/instance_manager.dart

```diff
@@ -0,0 +1,201 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
It looks like you're using the old `InstanceManager` implementation. I would recommend using the one from `wrapper_example`: https://github.com/bparrishMines/plugins/blob/wrapper_example/packages/wrapper_example/lib/src/instance_manager.dart. This also removes the need for `Copyable`.